app/v2/notifications/post_notifications.py

```diff
@@ -47,7 +47,7 @@ def post_notification(notification_type):
                                         notification_type=notification_type,
                                         api_key_id=api_user.id,
                                         key_type=api_user.key_type,
-                                        reference=form.get('reference', None),
+                                        client_reference=form.get('reference', None),
```

Comment:
if someone posted an email with a reference, and then we send it to SES, did we overwrite the user's reference with SES's one?